docs/resources/instance_server.md

```diff
@@ -176,6 +197,17 @@ attached to the server. Updates to this field will trigger a stop/start of the s
 - `project_id` - (Defaults to [provider](../index.md#project_id) `project_id`) The ID of the project the server is associated with.
 
 
+## Private Network
+
+~> **Important:** Updates to `private_network` will recreate the attachment Instance.
```

Comment:
What is an attachment Instance?

Reply:
attachement is recreate a new interface (Private NIC)

Reply:
Please check the new comment